aat/src/aat/resources/features/F-094 NoC ChallengeQuestions/F-094.feature

```diff
@@ -3,11 +3,11 @@ Feature: F-094 Retrieve ChallengeQuestions for Notice of Change
 
   Background:
     Given an appropriate test context as detailed in the test data source,
-    And a user [MCA System - with an IDAM role of caseworker CAA],
 
   @S-094.1
   Scenario: Must successfully return the contents of the ChallengeQuestion definition
-    Given the request [intends to get the contents of a set of questions],
+    Given a user [MCA System - with an IDAM role of caseworker CAA],
```

Comment:
do we need to mention about MCA System, as we are not depending on MCA. We can just say user with and IDAM role of caseworker CAA